RawSpeed/RawImage.cpp

```diff
@@ -337,8 +339,10 @@ void RawImageData::fixBadPixelsThread( int start_y, int end_y )
         for (int i = 0; i < 4; i++) {
           for (int j = 0; j < 8; j++) {
             if (1 == ((bad[i]>>j) & 1)) {
-              if (bad_count++ > 500)
+#ifdef __AFL_COMPILER
+              if (bad_count++ > 100)
```

Comment:
See the other comment, with RW2 there's no sensible limit that will work so I just enabled it only when running under afl and then made the limit smaller to be more effective.